one.mixin.messenger.yml

```diff
@@ -0,0 +1,45 @@
+id: one.mixin.messenger
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: mixin_desktop
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - name: mixin_desktop
+    buildsystem: simple
+    only-arches:
+    - x86_64
+    build-commands:
+    - mkdir -p FlutterApp
+    - tar -xf mixin_desktop_linux_amd64_portable.tar.gz -C FlutterApp
```

Comment:
I suggested using source type `archive`, with proper option, I guess this was totally ignored.

Reply:
I tried use archive as source type, but got an error when I try to do a local build. 
```
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

By use `-v` to check detailed build log, I found the error caused by `-z` parameter, but I don't known how to remove it...
```
tar xf /home/boyan/workspace/temp/flathub/.flatpak-builder/downloads/69ed7803b1c18bd4560e61abec727905ba5b7579cb6677e9865cf9034119d9e9/mixin_desktop_linux_amd64_portable.tar.gz --no-same-owner --strip-components=1 -z
```

Reply:
use `archive-type` to override it. But whatever is generating the tarball is doing it wrong.